src/common/interfaces/event.interface.ts

```diff
@@ -0,0 +1,223 @@
+import {
+  DirectoryUser,
+  DirectoryUserResponse,
+  EventDirectory,
+  EventDirectoryGroup,
+  EventDirectoryGroupResponse,
+  EventDirectoryResponse,
+} from '../../directory-sync/interfaces';
+import { Connection, ConnectionResponse } from '../../sso/interfaces';
+
+export interface EventBase {
```

Comment:
This file is largely just the existing `Webhook` types renamed to `Event` and moved to the common folder.